test/images/scripts/rhel-atomic.bootstrap

```diff
@@ -4,7 +4,7 @@
 set -e
 
 url="http://cdn.stage.redhat.com/content/dist/rhel/atomic/7/7Server/x86_64/images/"
-prefix="rhel-atomic-cloud-7.2"
+prefix="rhel-atomic-cloud-7.2-[0-9\.]+.x86_64.qcow2"
```

Comment:
I think this rhel atomic change would be better placed in its own pull request - together with an image update.
We'll probably want to switch to 7.3 soon (or now?).